featuretools/primitives/standard/transform/natural_language/median_word_length.py

```diff
@@ -16,13 +19,13 @@ class MedianWordLength(TransformPrimitive):
 
     Args:
         delimiters_regex (str): Delimiters as a regex string for splitting text into words.
-            The default delimiters include "- [].,!?;\\n".
+            The default delimiters include [-.!?]\n\t
 
     Examples:
         >>> x = ['This is a test file', 'This is second line', 'third line $1,000', None]
```

Comment:
this test was breaking up "$1,000" into "$1" and "000". After this change, it stays as one word